types/crypto-js/index.d.ts

```diff
@@ -30,6 +30,7 @@ declare namespace CryptoJS {
 		salt: string;
 		ciphertext: string;
 		key?: string;
+        toString(encoder?: Encoder): string;
```

Comment:
Next time please try to keep same indention in file. However it's not merge breaker. I accept it now.